org.eclipse.dawnsci.remotedataset.test/src/org/eclipse/dawnsci/remotedataset/test/ServiceHolder.java

```diff
@@ -21,13 +21,13 @@
 	public static IMarshallerService getMarshallerService() {
 		return marshaller;
 	}
-	public static void setMarshallerService(IMarshallerService marshaller) {
+	public void setMarshallerService(IMarshallerService marshaller) {
 		ServiceHolder.marshaller = marshaller;
 	}
 	public static ILoaderService getLoaderservice() {
 		return lservice;
 	}
-	public static void setLoaderService(ILoaderService lservice) {
+	public void setLoaderService(ILoaderService lservice) {
 		ServiceHolder.lservice = lservice;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)